src/UI/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs

```diff
@@ -132,7 +132,7 @@ internal class IndexModel<TUser> : IndexModel where TUser : IdentityUser
             var callbackUrl = Url.Page(
                 "/Account/ConfirmEmail",
                 pageHandler: null,
-                values: new { user.Id, code },
+                values: new { userId = user.Id, code = code },
```

Comment:
Was this a root cause of one of the bugs?

Reply:
Yes, the parameter being sent was not correct.